definitions/consented/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -1,2 +0,0 @@
-[
-]
```

Comment:
Thought the plan was to keep these empty non-prod files around so we don't have large PRs with unnecessary changes and tests being constantly updated each time. Doesn't matter either way but just something to consider

Reply:
we shouldn't use plain nonprod files anymore as we can't track these against an epic so I'm removing to avoid things being accidentally added in here

Reply:
Ah good shout 👍  Let's make sure to keep an eye on it going forward then in future reviews